pyproject.toml

```diff
@@ -12,8 +12,55 @@ addopts = [
 omit = [".env/*"]
 sort = "Cover"
 
-#[report]
-#sort = Cover
-#omit =
-#    .env/*
-#    backtracking/*
+[tool.codespell]
+ignore-words-list = "3rt,ans,crate,damon,fo,followings,hist,iff,kwanza,mater,secant,som,sur,tim,zar"
+skip = "./.*,*.json,ciphers/prehistoric_men.txt,project_euler/problem_022/p022_names.txt,pyproject.toml,strings/dictionary.txt,strings/words.txt"
+
+[tool.ruff]
+ignore = [
```

Comment:
It would be useful for posterity to specify why these rules are being ignored.

Reply:
Done.